legal/index.md

```diff
@@ -86,7 +86,7 @@ This website may collect personal information from some Users. Because The Moner
 
 ### Communications
 
-When you visit our website or send us email, our responses to your communications will be by e-mail. All agreements, notices and communications that we provide to you by email will satisfy all legal requirements that they be in writing and delivered to you. The Monero Project will have the right to disclose any email sent by a User, or any objectionable material, to comply with legal process or to protect the rights or property of The Monero Project, its customers, suppliers and Users.
+When you visit our website or send us an email, our responses to your communications will be by email. All agreements, notices and communications that we provide to you by email will satisfy all legal requirements that they be in writing and delivered to you. The Monero Project will have the right to disclose any email sent by a User, or any objectionable material, to comply with legal process or to protect the rights or property of The Monero Project, its customers, suppliers and Users.
```

Comment:
```suggestion
When you send us an email, our responses to your communications will be by email. All agreements, notices and communications that we provide to you by email will satisfy all legal requirements that they be in writing and delivered to you. The Monero Project will have the right to disclose any email sent by a User, or any objectionable material, to comply with legal process or to protect the rights or property of The Monero Project, its customers, suppliers and Users.
```

wen getmonero canary

Reply:
I find warrant canaries useless as Monero doesn't store any users PII like services that asks for email etc. Also, warrant canaries are useless as gag orders eliminates its use.

Reply:
I was joking